### PR TITLE
doc: update readme and docs to fit the current skipper project

### PR DESCRIPTION
**Pull request by szuecs**
doc: update readme and docs to fit the current skipper project

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: docs/data-clients/kubernetes.md [hunk 1]
```diff
@@ -22,32 +21,35 @@ proxy made to apply updates very often. Skipper is used in
...
-  - [traffic by percentage](https://godoc.org/github.com/zalando/skipper/predicates/traffic) supports also sticky sessions
+   - [collection of base http manipulations](../reference/filters.md):
+     for example [manipulating Path](../reference/filters.md#http-path), [Querystring](../reference/filters.md#http-query), [HTTP Headers](../reference/filters.md#http-headers) and [redirect](../reference/filters.md#http-redirect) handling
+   - [cookie handling](../reference/filters.md#cookie-handling)
+   - [circuitbreakers](../reference/filters.md#circuit-breakers)
+   - [ratelimit](../reference/filters.md#rate-limit): based on client or backend data
+   - [Shadow traffic filters](../reference/filters.md#shadow-traffic)
+- [Predicates](../reference/predicates.md) - advanced matching capability
+   - URL [Path](../reference/predicates.md#the-path-tree) match: `Path("/foo")`
+   - Host header match: `Host("^www.example.org$")`
```
Comment (RomanZavodskikh) [inline docs/data-clients/kubernetes.md:40]: We can add link to the Host predicate as well.
https://opensource.zalando.com/skipper/reference/predicates/#host

File: docs/data-clients/kubernetes.md [hunk 2]
```diff
@@ -22,32 +21,35 @@ proxy made to apply updates very often. Skipper is used in
...
+   - [traffic by percentage](../reference/predicates.md#trafficsegment) supports also sticky sessions
 - Kubernetes integration
-  - All Filters and Predicates can be used with 2 annotations
-    - Predicates: `zalando.org/skipper-predicate`
-    - Filters: `zalando.org/skipper-filter`
-  - Custom routes can be defined with the annotation `zalando.org/skipper-routes`
-  - [metrics](https://godoc.org/github.com/zalando/skipper/metrics)
-  - access logs
-  - Blue-Green deployments, with another Ingress annotation `zalando.org/backend-weights`
+   - All Filters and Predicates can be used with 2 annotations
```
Comment (RomanZavodskikh) [inline docs/data-clients/kubernetes.md:47]: We can add link about annotations here:
https://opensource.zalando.com/skipper/kubernetes/ingress-usage/#skipper-ingress-annotations

File: readme.md [hunk 3]
```diff
@@ -182,20 +191,25 @@ includes information about [deployment use cases](https://opensource.zalando.com
...
+- [Proxy Package](https://pkg.go.dev/github.com/zalando/skipper/proxy)
 - [Logging](https://pkg.go.dev/github.com/zalando/skipper/logging) and [Metrics](https://pkg.go.dev/github.com/zalando/skipper/metrics)
-- Performance Considerations
+- [Operations guide](https://opensource.zalando.com/skipper/operation/operation/)
+- [Authentication and Authorization](https://opensource.zalando.com/skipper/reference/filters/#authentication-and-authorization)
+- [Load Shedders](https://opensource.zalando.com/skipper/reference/filters/#load-shedding)
 - [Rate Limiters](https://pkg.go.dev/github.com/zalando/skipper/filters/ratelimit)
-- [Opentracing plugin](https://github.com/skipper-plugins/opentracing/) or extend [create your own](https://opensource.zalando.com/skipper/reference/plugins/#opentracing-plugins)
-- [WAF plugin](https://github.com/jptosso/coraza-waf) - Web Application Firewall filter for Skipper routes
```
Comment (RomanZavodskikh) [inline readme.md:198]: Why do we remove WAF plugin?

Comment (szuecs) [reply readme.md:198]: Because it changed to not depend on skipper anymore

File: docs/data-clients/kubernetes.md (hunk 2 above)
Comment (szuecs) [reply docs/data-clients/kubernetes.md:47]: done

File: docs/data-clients/kubernetes.md (hunk 1 above)
Comment (szuecs) [reply docs/data-clients/kubernetes.md:40]: done

**Comment by szuecs** [comment]
:+1:

**Pull request merged**